src/java.base/share/classes/java/io/StringReader.java

```diff
@@ -30,31 +30,28 @@
 /**
  * A character stream whose source is a string.
  *
+ * @apiNote
+ * As of release JDK 24, this class has been supplemented with an equivalent
+ * API designed for use by a single thread, {@link Reader#of(CharSequence)}.
+ * {@code Reader.of(String)} should generally be used in preference to this one,
+ * as it supports all of the same operations but it is faster, as it performs no
+ * synchronization.
```

Comment:
We shouldn't using "release 24" or "supplemented" here. The API note just needs to `Reader.of(CharSequence)` provides a method to read from any CharSequence that may be more efficient than `StringReader`.

Reply:
I think "release 24" is fine for now as similar text exists in `java.util.Vector`. We can normalize such texts when we implement https://bugs.openjdk.org/browse/JDK-8341736.

Reply:
Please remove the reference to JDK 24; it will age rapidly and would require extra maintenance later.
The javadoc should focus on the functionality provided.

Reply:
Done; replaced by Alan's proposal: "Reader.of(CharSequence) provides a method to read from any CharSequence that may be more efficient than StringReader"